app/services/container_dashboard_service.rb

```diff
@@ -175,8 +173,54 @@ def ems_utilization
           :yData => ["used"] + used_mem.values.map { |m| (m / 1024.0).round }
         }
       }
-    else
-      {}
     end
   end
+
+  def hourly_network_trends
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    hourly_network_trend = Hash.new(0)
+    hourly_metrics =
+      MetricRollup.find_all_by_interval_and_time_range("hourly", 24.hours.ago.beginning_of_hour.utc, Time.now.utc)
+    hourly_metrics =
+      hourly_metrics.where('resource_type = ? AND resource_id in (?)', 'ExtManagementSystem', resource_ids)
```

Comment:
@zeari Using pluck is really recommended when you need the actual array of ids, its better preference wise because it just returns an array of results and doesn't instantiate an array of objects (select does), but pluck is an eager method that hits the database immediately when called, in this case you don't need the actual array (just using it for the query) so just using select saves you a query because it won't get called until needed.
In general use pluck for stand alone queries.